src/pages/tutorials/backend/graphql-js/3-mutations.md

```diff
@@ -2,5 +2,50 @@
 title: Mutations
 ---
 
-- resolvers with arguments
-- creating links
\ No newline at end of file
+### Mutation for creating links
+
+Setting up mutations is as easy as queries, following the same process.
+
+<Instruction>
+
+First, add the `createLink` mutation to that `typeDefs` variable in `src/schema/index.js`:
+
+```graphql(path=".../hackernews-graphql-js/src/schema/index.js")
+type Mutation {
+  createLink(url: String!, description: String!): Link
+}
+```
+
+</Instruction>
+
+### Resolvers with arguments
+
+<Instruction>
+
+Now add a resolver for `createLink` inside `src/schema/resolvers.js:`
+
+```js{5-11}(path=".../hackernews-graphql-js/src/schema/resolvers.js")
+module.exports = {
+  Query: {
+    allLinks: () => links,
+  },
+  Mutation: {
+    createLink: (_, data) => {
+      const newLink = Object.assign({id: links.length}, data);
```

Comment:
Thanks, good point! Fixing this here: https://github.com/howtographql/howtographql/pull/130.